source/Ship.cpp

```diff
@@ -2064,6 +2067,9 @@ void Ship::Fire(vector<Projectile> &projectiles, vector<Visual> &visuals)
 				}
 			}
 		}
+		else if(isFlagship && canFire == CanFireResult::NO_AMMO
```

Comment:
Should a lack of other resources also trigger a sound? Flamethrower with insufficient fuel, for example.